custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -123,6 +120,27 @@ def update_query(self):
             'date': self.date
         }
 
+    def update_launched_query(self):
+        return """
+        UPDATE "{tablename}" agg_awc SET
+           num_launched_states = CASE WHEN ut.all_cases_household>0 THEN 1 ELSE 0 END,
```

Comment:
along with these 5 columns can you also do this for cases_household` just fo make the data consistent.

Reply:
Done, Please review.